src/conda_smithy.rst

```diff
@@ -41,6 +41,14 @@ conda smithy rerender
 
 and commit the changes!
 
+Optionally one can commit the changes automatically.
+To do this just use the `--commit`/`-c` option. By default,
+this will open an editor to make a commit. It will provide
```

Comment:
Not sure if it matters, but this could be interpreted as smithy opening the editor, rather than git doing its own default behavior. One option for that is something like:

> By default, git will open an editor to write the commit message.